### DISCUSSION TITLE
How to install sample sets in Grand Orgue.

### DISCUSSION THREAD

**Discussion by larspalo** [GrandOrgue/grandorgue#370] [General]
Originally posted 2021-02-28 19:11:27.607000 by bakfiets in GrandOrgue discussion forum on SourceForge<br><br>Just beginning with Grand Orgue and first thing I considered was: install a sampleset, with the bar ( left button (Bestand (dutch) or File (eng.)) which  is to be found on the top of the screen. I need something with
.orgue .... and don&#039;t find this in de folder: many folders like: GO, PED, POS and than the logo of GrandOrgue with the name of the organ. What next to do with these folders? Thanks ahead a lot.

**Comment by larspalo** [comment]
Originally posted 2021-03-01 08:38:34.352000 by [csw900](https://github.com/midimusic) in GrandOrgue discussion forum on SourceForge<br><br>I agree it is confusing - I have been using GO for years and still do not fully understand all of it. Try looking for an  &quot;.organ&quot;  file in your sampleset and double click it. GO should start and the organ should load.

csw900

**Reply by larspalo** [reply]
Originally posted 2021-03-01 11:49:04.605000 by bakfiets in GrandOrgue discussion forum on SourceForge<br><br>.. ok, thanks, but I succeeded: I just pressed on the logo of G. Or. ( 3
white/black bars) in the folder there: it acts like .exe! And the organ is
visible in the display of G. O.


* mei freonlike groet,*

*  met vriendelijke groet, *


*  Jitske Hibma *

*  de Kampen 18*
*  9132 LV
&lt;https://docs.google.com/document/d/1e1Ug9mq6mS04ALLrgSvfYQQsMcLPRS7gpYNOrygwo84/edit?usp=sharing&gt;*
*Engwierum*
*  ----------------  *

*  www.veldfiets.nl &lt;http://www.veldfiets.nl&gt; *







Op ma 1 mrt. 2021 om 09:38 schreef csw900 &lt;csw900@users.sourceforge.net&gt;:

&gt; I agree it is confusing - I have been using GO for years and still do not
&gt; fully understand all of it. Try looking for an &quot;.organ&quot; file in your
&gt; sampleset and double click it. GO should start and the organ should load.
&gt;
&gt; csw900
&gt; ------------------------------
&gt;
&gt; How to install sample sets in Grand Orgue.
&gt; &lt;https://sourceforge.net/p/ourorgan/discussion/962124/thread/4f5b1763af/?limit=25#6ca8&gt;
&gt; ------------------------------
&gt;
&gt; Sent from sourceforge.net because you indicated interest in
&gt; https://sourceforge.net/p/ourorgan/discussion/962124/
&gt;
&gt; To unsubscribe from further messages, please visit
&gt; https://sourceforge.net/auth/subscriptions/
&gt;
<br><br>Attachments in original post:<br>https://sourceforge.net/p/ourorgan/discussion/962124/thread/4f5b1763af/6ca8/5c43/attachment/alternate<br>

**Comment by larspalo** [comment]
Originally posted 2021-03-01 09:04:44.814000 by [Lars Palo](https://github.com/larspalo) in GrandOrgue discussion forum on SourceForge<br><br>Wellcome to the wonderful world of GrandOrgue!

If you&#039;re really new to GO, please note that you&#039;ll need one (or more)
sampleset(s) and not only the GO software. Download what you want and need.

Also note that there are different versions of samplesets that require
slightly different approaches when it comes to opening/loading/installing
them. Basically, the old formats where you open an .organ file and the
newer format where you install an .orgue file (or simply drop the file in
the appropriate folder). You cannot do the other way around (easily).
<br><br>Attachments in original post:<br>https://sourceforge.net/p/ourorgan/discussion/962124/thread/4f5b1763af/7174/7fea/attachment/alternate<br>

**Comment by oleg68** [comment]
> same result v 3.13 'quit unexpectedly' 'reopen/report/ignore'
> 
> on iMac osx 12.5

Is it x86_64, m1 or m2?

**Reply by oleg68** [reply]
Could you clarify, what sample set were you opening exactly when you received this crash dump?

**Reply by oleg68** [reply]
I added some diagnostic to [the intermediate build](https://github.com/oleg68/GrandOrgue-official/suites/17728402655/artifacts/1015045488). Could you try to run it?

**Reply by oleg68** [reply]
Please check the sizes of your files in the sample set.

For example, the file `Kalvtrask_Church/Principal8/039-D#.wav` should have 1751314 bytes

```
[oleg@oleg2 sample_sets]$ ls -l Kalvtrask_Church/Principal8/039-D#.wav
-rw-r--r--. 1 oleg users 1751314 Mar  5  2012 Kalvtrask_Church/Principal8/039-D#.wav
```

What is the actual size of this file on your computer?

**Reply by larspalo** [reply]
@pau142 Could it be due to download issues? Have you compared the md5sum of the downloaded .rar archive with the numbers I've published on http://familjenpalo.se/vpo/download/ to rule out some kind of corruption?

**Reply by oleg68** [reply]
@pau142 it looks like your unpacker truncates files, so the samples become damaged.
